aslam_cv_calibration/test/test-target-observation.cc

```diff
@@ -93,19 +93,19 @@ class TargetObservationTest : public ::testing::Test {
 
   void corruptWithGaussianNoise(
       const Eigen::Matrix2Xd& image_points,
-      Eigen::Matrix2Xd* image_points_corruped) {
+      Eigen::Matrix2Xd* image_points_corrupted) {
     const size_t num_rows = image_points.rows();
     CHECK_EQ(num_rows, 2u);
     const size_t num_cols = image_points.cols();
-    *CHECK_NOTNULL(image_points_corruped) = image_points;
+    *CHECK_NOTNULL(image_points_corrupted) = image_points;
     std::random_device random_device{};
     std::mt19937 random_generator{random_device()};
     constexpr double kStandardDeviationPixels = 0.5;
     std::normal_distribution<> normal_distribution{0.0,
                                                    kStandardDeviationPixels};
-    for (size_t col_index = 0u; col_index < num_cols; ++col_index) {
-      for (size_t row_index = 0u; row_index < num_rows; ++row_index) {
-        (*image_points_corruped)(col_index, row_index) +=
+    for (size_t row_index = 0u; row_index < num_rows; ++row_index) {
+      for (size_t col_index = 0u; col_index < num_cols; ++col_index) {
+        (*image_points_corrupted)(row_index, col_index) +=
```

Comment:
@schneith The bug was actually here (interchanging of row/col index), not sure why this was actually working :-)